src/components/ColorPicker/__stories__/ColorPicker.stories.mdx

```diff
@@ -4,13 +4,18 @@ import { createComponentTemplate, createStoryMetaSettings } from "../../../story
 import TextColorIndicator from "../../Icon/Icons/components/TextColorIndicator";
 import Check from "../../Icon/Icons/components/Check";
 
-export const argTypes = createStoryMetaSettings({
+export const metaSettings = createStoryMetaSettings({
   component: ColorPicker,
   enumPropNamesArray: ["colorStyle", "colorSize"],
-  actionPropsArray: ["onSave"]
+  actionPropsArray: [{name: 'onSave', linkToValueProp: 'value'}]
```

Comment:
I think linter should warn about using different " / ' styles in the same page

Reply:
by looking at `eslintrc.js` it looks like the rule is disabled
![image](https://user-images.githubusercontent.com/96776835/149760103-a87403a1-022d-439e-8d3f-a4a2ca84cffd.png)
do you think we should change it?

regardless, I'll fix this file :)